lib/rmt/downloader.rb

```diff
@@ -72,7 +58,7 @@ def process_queue
       begin
         make_request(remote_file, local_file, request_fiber, queue_item[:checksum_type], queue_item[:checksum])
       rescue RMT::Downloader::Exception => e
-        @logger.info("E #{File.basename(local_file)} - #{e}")
+        @logger.warn("× #{File.basename(local_file)} - #{e}")
```

Comment:
It's a multiplication sign, it should be in base ASCII charset. But yeah, it looks unorthodox :-)